components/Cms/Block/CmsBlockGalleryBuybox.vue

```diff
@@ -9,6 +9,25 @@ const { getSlotContent } = useCmsBlock(props.block);
 
 const leftContent: Schemas['CmsSlot'] = getSlotContent('left');
 const rightContent: Schemas['CmsSlot'] = getSlotContent('right');
+
+const productData = inject('productData');
+const product = productData.value;
```

Comment:
```suggestion
const product = inject('productData');
```

or why are we saving it in another variable?

Reply:
I saved it in another variable so that in the following lines I wont have to always call `product.value` when accessing it (happens 4 times). We can change it of course, both would be fine for me

Reply:
ah okay! i think we should stick with product.value because otherwise we would lost the reactivity, right?

Reply:
I changed it to use .value instead